Contributing.MD

```diff
@@ -0,0 +1,30 @@
+Contributing to README Template
+
+We welcome your contributions to README Template. This doc describes the process to contribute patches to README Template and the general guidelines we expect contributors to follow.
```

Comment:
I like how [Code.gov's CONTRIBUTING file ](https://github.com/GSA/code-gov-front-end/blob/master/CONTRIBUTING.md) lays out multiple ways users can submit issues, including creating GitHub Issues, emailing their email inbox, and creating pull requests. I think we should add something like that in here.

Reply:
I also like Code.gov's Welcome! section:

We're so glad you're thinking about contributing to a U.S. Government open source project! If you're unsure about anything, just ask -- or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.